webhook/runtime_webhooks.md

```diff
@@ -0,0 +1,103 @@
+
+## Runtime Actions as Webhook
+
+Adobe I/O Runtime lets you create actions and set them up as webhook to incorporate custom responses to events in your applications. You can either create synchronous actions and use them as events integration webhook or for long-running async actions you can use the [journaling approach](https://adobeio-codelabs-journaling-events-adobedocs.hlx.page/?src=/README.html) for consuming events.
```

Comment:
if you can call out this document is meant for webhook only not journaling, then would help the reader understand the overall context.